relayer/cmd/getblock.go

```diff
@@ -72,11 +85,16 @@ func getEthBlock(config *ethereum.Config) (*gethTypes.Header, error) {
 		"chainID":  chainID,
 	}).Info("Connected to chain")
 
-	latestHeader, err := client.HeaderByNumber(ctx, nil)
-	header, err := client.HeaderByNumber(
-		ctx,
-		new(big.Int).Sub(latestHeader.Number, big.NewInt(int64(config.DescendantsUntilFinal))),
-	)
+	var header *gethTypes.Header
+	if blockHash == nil {
+		header, err = client.HeaderByNumber(ctx, nil)
```

Comment:
```suggestion
```

Reply:
I'm using header.Number right below to calculate the finalized block number. Admittedly I prob shouldn't be re-using the variable

Reply:
oh right my bad - but ye maybe use different names :)